apps/docs/components/motionPreview/MotionPreview.css

```diff
@@ -0,0 +1,61 @@
+.hd-motion-preview {
+    margin-bottom: var(--hd-space-4);
+    gap: var(--hd-space-5);
+    align-items: flex-end;
+    display: flex;
+    flex-flow: wrap;
+}
+
+.hd-object {
+    background-color: #6B5B7B;
+    width: 10rem;
+    height: 10rem;
+    transition: transform var(--hd-duration) var(--hd-easing);
+}
+
+.hd-object-animated {
+    transform: translateX(6.25rem);
+}
+
```

Comment:
overall terrible CSS, didnt want to spend too much time customizing react aria components, since we,ll probably use something else at some point